indico/modules/events/abstracts/controllers/email_templates.py

```diff
@@ -0,0 +1,119 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import request
+
+from indico.core.db import db
+from indico.modules.events.abstracts.controllers.management import RHManageAbstractsBase
+from indico.modules.events.abstracts.forms import (EmailTemplateRuleFormEdit, EmailTemplateTextFormEdit,
+                                                   EmailTemplateFormCreate)
+from indico.modules.events.abstracts.models.email_templates import AbstractEmailTemplate
+from indico.modules.events.abstracts.util import build_default_email_template, create_mock_abstract
+from indico.util.placeholders import replace_placeholders
+from indico.web.flask.templating import get_template_module
+from indico.web.util import jsonify_data, jsonify_template
+
+
+class RHEmailTemplateList(RHManageAbstractsBase):
+    """Display list of e-mail templates."""
+
+    def _process(self):
+        return jsonify_template('events/abstracts/management/notification_tpl_list.html', event=self.event_new,
+                                email_tpls=list(self.event_new.abstract_email_templates))
```

Comment:
What about changing the `lazy='dynamic'` to `lazy=True` on that relationship? I can't see any good reason for dynamic there